FirstSolution/Part9Task1/Task.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Part9Task1
+{
+    // Enter the desired length line from the keyboard and calculate
+    // the percentage of the character entries in the string.
+    
+    public static class Task
+    {
+        private static void Main(string[] args)
+        {
+            string text = Console.ReadLine();
+            Console.WriteLine("Enter the character");
+            char symbol = Convert.ToChar(Console.ReadLine());
+            double num = 0;
+            
+            if (text != null)
+            {
```

Comment:
Add check before loop body and throw exception

Reply:
Done